### PR TITLE
Use new CalcSpatialInertia() function in manipulation_station

### PR DESCRIPTION
**Pull request by mitiguy**
Test new calculations in manipulation_station.cc that employ CalcSpatialInertia() against old function MakeCompositeGripperInertia() that previously was in manipulation_station.cc and has now been moved to manipulation_station_test.cc.

- - -
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)

### CODE REVIEW

**Review by mitiguy — COMMENTED** [review]
Feature review +@sherm1 

> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847#-:-NBKS_9T-rYsxzylBEtt:bwuh3l8)* status: LGTM missing from assignee sherm1(platform), needs at least two assigned reviewers, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits), missing label for release notes (waiting on @sherm1)

**Review by sherm1 — COMMENTED** [review]
Nice -- feature :lgtm: pending a few nits
+@xuchenhan-tri for platform review per rotation, please

> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847#-:-NBOhPQc-eeLGKy48QAA:b-nwmnl6)* status: 10 unresolved discussions, LGTM missing from assignee xuchenhan-tri(platform), commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @mitiguy and @xuchenhan-tri)

___
*[`examples/manipulation_station/manipulation_station.h` line 489 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOe2gIB9dOsDMxZgsU:-NBOe2gIB9dOsDMxZgsV:b-ba61q9) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L489)):*
> ```objc
>   /// Returns the child frame name for the controller model.
>   /// @throws std::exception if Finalize() has not been called.
>   std::string controller_model_child_frame_name() {
> ```

BTW consider prefacing the comments here with "(Internal use only)" if they don't have utility beyond testing. You could alternatively exempt them from Doxygen so no one will know they are there.
___
*[`examples/manipulation_station/manipulation_station.cc` line 59 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOeoxX7V-SN1bDmOMo:-NBOeoxX7V-SN1bDmOMp:b7ll99a) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.cc#L59)):*
> ```cpp
> // @param[in] gripper_body_frame_name Name of the frame attached to the
> //            gripper's main body.
> // @retval M_SGo_G, spatial inertia of set S about Go, expressed in frame G.
> ```

nit: should be no comma after the `@retval` name (just like `@param` above)
___
*[`examples/manipulation_station/manipulation_station.cc` line 71 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOf6LW7PZEoPHmut0r:-NBOf6LW7PZEoPHmut0s:b-9iorjb) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.cc#L71)):*
> ```cpp
>   // Set bodies to have zero translation, zero rotation, zero velocity, etc.
>   auto context = plant.CreateDefaultContext();
>   plant.SetDefaultState(*context, &context->get_mutable_state());
> ```

BTW is this needed? I would have expected the default context to contain a default state.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 39 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOfrpTCtmduFmow5Y6:-NBOfrpTCtmduFmow5Y7:b4ylplk) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L39)):*
> ```cpp
> // @param[in] gripper_body_frame_name Name of the frame attached to the
> //            gripper's main body.
> // @retval M_SGo_G, spatial inertia of set S about Go, expressed in frame G.
> ```

nit: should be no comma after the retval name
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 42 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOflu0DkPmSR0w21Wj:-NBOflu0DkPmSR0w21Wk:b-lapvur) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L42)):*
> ```cpp
> // @note Previously, this function was in manipulation_station.cc and calculated
> //   set S's spatial inertia. Now, it only verifies CalcGripperSpatialInertia()
> //   which in an internal namespace in manipulation_station.cc.
> ```

typo: which _is_ in
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 63 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOg18w1U7i4kbONfQ5:-NBOg18w1U7i4kbONfQ6:b-h3tyxv) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L63)):*
> ```cpp
>   const multibody::SpatialInertia<double>& M_RRo_R =
>       right_finger.default_spatial_inertia();
>   auto CalcFingerPoseInGripperFrame =
> ```

nit: per GSG variable names must use `snake_case` (even though the value is a lambda function it is still a variable)
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 82 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOgWk97zEx49HwcJZM:-NBOgWk97zEx49HwcJZN:b2twe4n) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L82)):*
> ```cpp
>   // Pose of right finger R in gripper frame G when the slider's dof is zero.
>   const RigidTransform<double> X_GR(CalcFingerPoseInGripperFrame(right_slider));
>   // Helper to compute the spatial inertia of a finger F in about the gripper's
> ```

typo: ~in~ about -> about
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 84 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOgdLp5T1UcA2iX2vE:-NBOgdLqC4NBGFU1cnYI:b-atnhje) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L84)):*
> ```cpp
>   // Helper to compute the spatial inertia of a finger F in about the gripper's
>   // origin Go, expressed in G.
>   auto CalcFingerSpatialInertiaInGripperFrame =
> ```

nit: snake_case required
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 210 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOguE33F86zosKEjEa:-NBOguE33F86zosKEjEb:b-ssqdbm) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L210)):*
> ```cpp
>   // in G, where the fingers are regarded as being in a "zero" configuration.
>   // Verify the spatial inertia stored in the controller_plant matches the
>   // calculation returned by MakeCompoiseGripperInertia().
> ```

typo: Compoise -> Composite
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 224 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOh3weEMmRg-Nzvi_r:-NBOh3weEMmRg-Nzvi_s:b-k6d9px) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L224)):*
> ```cpp
>   const Matrix6<double> M6_actual = M_SGo_G_actual.CopyToFullMatrix6();
>   const Matrix6<double> M6_expected = M_SGo_G_expected.CopyToFullMatrix6();
>   constexpr double kEpsilon = 32 * std::numeric_limits<double>::epsilon();
> ```

nit: rename this something like `kTol` -- it is misleading to call it "epsilon" since it has a different value than machine epsilon

**Review by jwnimmer-tri — COMMENTED** [review]
> Reviewed 1 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: 10 unresolved discussions, LGTM missing from assignee xuchenhan-tri(platform), commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @mitiguy, @sherm1, and @xuchenhan-tri)

___
*[`examples/manipulation_station/manipulation_station.h` line 489 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOe2gIB9dOsDMxZgsU:-NBOhx9I-ekn3wnYmQhX:b-g70xoh) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L489)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

BTW consider prefacing the comments here with "(Internal use only)" if they don't have utility beyond testing. You could alternatively exempt them from Doxygen so no one will know they are there.
</blockquote></details>

There is no need to expose these for unit testing.  (The unit test can hard-code the model names just fine, it doesn't need to ask this class for what they are.)  This pollutes the header file for no good reason.

**Review by xuchenhan-tri — COMMENTED** [review]
platform :lgtm: pending resolution of existing threads.

> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847#-:-NBOtwfH64u_XmzLa6OQ:b-16ghkg)* status: 13 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @jwnimmer-tri and @mitiguy)

___
*[`examples/manipulation_station/manipulation_station.h` line 482 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOo9pq7rzfRJhXNF99:-NBOo9pq7rzfRJhXNF9A:b-ghg0vj) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L482)):*
> ```objc
>   /// Returns the path to the sdf file that contains the controller model.
>   /// @throws std::exception if Finalize() has not been called.
>   std::string controller_model_path() {
> ```

nit this should be const.
___
*[`examples/manipulation_station/manipulation_station.h` line 489 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOe2gIB9dOsDMxZgsU:-NBOoOtY7H6E5GaemJlh:b-vwvqbw) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L489)):*
<details><summary><i>Previously, jwnimmer-tri (Jeremy Nimmer) wrote…</i></summary><blockquote>

There is no need to expose these for unit testing.  (The unit test can hard-code the model names just fine, it doesn't need to ask this class for what they are.)  This pollutes the header file for no good reason.
</blockquote></details>

Looks like this function and the one above are used as helpers in `ManipulationStation<T>::MakeIiwaControllerModel()`. In that case, they can be private.

In any case, they should be const and should return by const reference.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 28 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOqNd63C38mHYTPSYA:-NBOqNd63C38mHYTPSYB:bf2cty3) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L28)):*
> ```cpp
> using systems::BasicVector;
> ```

nit extraneous blank line. Also, is the internal namespace needed? This is already in an unnamed namespace.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 40 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOsr3h1cWoSa8GBVf5:-NBOsr3h1cWoSa8GBVf6:b-cq1r1n) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L40)):*
> ```cpp
> //            gripper's main body.
> // @retval M_SGo_G, spatial inertia of set S about Go, expressed in frame G.
> // @note Previously, this function was in manipulation_station.cc and calculated
> ```

minor, IMO this note provides more value if it simply states this is an alternative way to calculate the spatial inertia of the gripper to unit test `CalcGripperSpatialInertia`. 

The previous location of this code and the namespace in which CalcGripperSpatialInertia exists is distracting and would have prevented me from understanding why this function is here had I not read the commit message.

**Review by sherm1 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: 13 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @jwnimmer-tri and @mitiguy)

___
*[`examples/manipulation_station/manipulation_station.h` line 489 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOe2gIB9dOsDMxZgsU:-NBP5VRA-4AbujgpsIYZ:b-n2i3t8) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L489)):*
> The unit test can hard-code the model names

I agree this would be better if we can count on the path name being the same on all CI platforms.

**Review by mitiguy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: 2 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @jwnimmer-tri, @sherm1, and @xuchenhan-tri)

___
*[`examples/manipulation_station/manipulation_station.h` line 482 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOo9pq7rzfRJhXNF99:-NBThz7s0g9zmiHzVcwZ:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L482)):*
<details><summary><i>Previously, xuchenhan-tri wrote…</i></summary><blockquote>

nit this should be const.
</blockquote></details>

Done.
___
*[`examples/manipulation_station/manipulation_station.h` line 489 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOe2gIB9dOsDMxZgsU:-NBTyT9PDskKxQxfwau_:bqjqi8s) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L489)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

> The unit test can hard-code the model names

I agree this would be better if we can count on the path name being the same on all CI platforms.
</blockquote></details>

Done (maybe).  From what I can tell, the controller_model_child_frame_name() might make sense to hard code as its value is simply "body"

The controller_model_path() is more suspect as this is its value on my computer:
/home/mitiguy/.cache/bazel/_bazel_mitiguy/d8ed8bf761808cb3cfbf4393f5d7c6d5/execroot/drake/bazel-out/k8-dbg/bin/examples/manipulation_station/manipulation_station_test.runfiles/drake/manipulation/models/wsg_50_description/sdf/schunk_wsg_50_no_tip.sdf
___
*[`examples/manipulation_station/manipulation_station.cc` line 59 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOeoxX7V-SN1bDmOMo:-NBTibUTB8he3TwpsOm3:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.cc#L59)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

nit: should be no comma after the `@retval` name (just like `@param` above)
</blockquote></details>

Done.
___
*[`examples/manipulation_station/manipulation_station.cc` line 71 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOf6LW7PZEoPHmut0r:-NBTx9-uEm37dvGZr_X7:b-1wyfw4) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.cc#L71)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

BTW is this needed? I would have expected the default context to contain a default state.
</blockquote></details>

Done.  Updated comment accordingly.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 28 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOqNd63C38mHYTPSYA:-NBTjUgz6lY8by-zdVra:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L28)):*
<details><summary><i>Previously, xuchenhan-tri wrote…</i></summary><blockquote>

nit extraneous blank line. Also, is the internal namespace needed? This is already in an unnamed namespace.
</blockquote></details>

Done.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 39 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOfrpTCtmduFmow5Y6:-NBTjW1qAxz3cDvoZBNn:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L39)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

nit: should be no comma after the retval name
</blockquote></details>

Done.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 40 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOsr3h1cWoSa8GBVf5:-NBTkpGM-h4Vj5KVg5Hu:bs7wumb) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L40)):*
<details><summary><i>Previously, xuchenhan-tri wrote…</i></summary><blockquote>

minor, IMO this note provides more value if it simply states this is an alternative way to calculate the spatial inertia of the gripper to unit test `CalcGripperSpatialInertia`. 

The previous location of this code and the namespace in which CalcGripperSpatialInertia exists is distracting and would have prevented me from understanding why this function is here had I not read the commit message.
</blockquote></details>

Done.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 42 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOflu0DkPmSR0w21Wj:-NBTkx3EEY0URSwInOol:b-qj92pf) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L42)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

typo: which _is_ in
</blockquote></details>

Reworded.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 63 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOg18w1U7i4kbONfQ5:-NBTldsKCP6g_2MM3lO-:b3rfxa5) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L63)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

nit: per GSG variable names must use `snake_case` (even though the value is a lambda function it is still a variable)
</blockquote></details>

Done.  FYI: Code was previously in master and cut-paste from manipulation_station.cc.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 82 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOgWk97zEx49HwcJZM:-NBTllSN1rVfTN0QS8hj:b-pfig7n) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L82)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

typo: ~in~ about -> about
</blockquote></details>

Done.  FYI: Code was previously in master and cut-paste from manipulation_station.cc.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 84 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOgdLp5T1UcA2iX2vE:-NBTmBwi63F_GDLX7uPA:b3rfxa5) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L84)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

nit: snake_case required
</blockquote></details>

Done.  FYI: Code was previously in master and cut-paste from manipulation_station.cc.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 210 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOguE33F86zosKEjEa:-NBTmTN32-ZcFkJwmcV_:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L210)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

typo: Compoise -> Composite
</blockquote></details>

Done.
___
*[`examples/manipulation_station/test/manipulation_station_test.cc` line 224 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOh3weEMmRg-Nzvi_r:-NBTvP1f7ZUK-YOucqs-:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/test/manipulation_station_test.cc#L224)):*
<details><summary><i>Previously, sherm1 (Michael Sherman) wrote…</i></summary><blockquote>

nit: rename this something like `kTol` -- it is misleading to call it "epsilon" since it has a different value than machine epsilon
</blockquote></details>

Done.

**Review by jwnimmer-tri — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: 2 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @jwnimmer-tri, @sherm1, and @xuchenhan-tri)

___
*[`examples/manipulation_station/manipulation_station.h` line 489 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOe2gIB9dOsDMxZgsU:-NBUCMk_3TyOljs--8dS:b-qs4w43) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L489)):*
<details><summary><i>Previously, mitiguy (Mitiguy) wrote…</i></summary><blockquote>

Done (maybe).  From what I can tell, the controller_model_child_frame_name() might make sense to hard code as its value is simply "body"

The controller_model_path() is more suspect as this is its value on my computer:
/home/mitiguy/.cache/bazel/_bazel_mitiguy/d8ed8bf761808cb3cfbf4393f5d7c6d5/execroot/drake/bazel-out/k8-dbg/bin/examples/manipulation_station/manipulation_station_test.runfiles/drake/manipulation/models/wsg_50_description/sdf/schunk_wsg_50_no_tip.sdf
</blockquote></details>

The string value to hard-code in the test would be `"drake/manipulation/models/wsg_50_description/sdf/schunk_wsg_50_no_tip.sdf"`.  To turn that into absolute path, call `FindResourceOrThrow`.

In other words, replicating the first switch-case item within `AddDefaultWsg` from within the implementation.

**Review by jwnimmer-tri — COMMENTED** [review]
> Reviewed 1 of 3 files at r2.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: 2 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @sherm1 and @xuchenhan-tri)

**Review by xuchenhan-tri — COMMENTED** [review]
> Reviewed 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: 2 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @sherm1)

___
*[`examples/manipulation_station/manipulation_station.h` line 482 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOo9pq7rzfRJhXNF99:-NBUKTgDCu34kKHwuw4k:b-edfxk5) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L482)):*
<details><summary><i>Previously, mitiguy (Mitiguy) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Make it a const member function too (a moot point if you remove this function)



_Suggestion:_
```objc
const std::string& controller_model_path() const {
```

**Review by xuchenhan-tri — COMMENTED** [review]
> Reviewed 3 of 3 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: 2 unresolved discussions, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @sherm1)

**Review by jwnimmer-tri — COMMENTED** [review]
> Reviewed 1 of 3 files at r3.
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: 1 unresolved discussion, commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @sherm1)

**Review by mitiguy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847)* status: commits need curation (https://drake.mit.edu/reviewable.html#curated-commits) (waiting on @mitiguy)

___
*[`examples/manipulation_station/manipulation_station.h` line 482 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOo9pq7rzfRJhXNF99:-NBUeytx5Hc0S3R-qxrl:b-xv3osf) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L482)):*
<details><summary><i>Previously, xuchenhan-tri wrote…</i></summary><blockquote>

Make it a const member function too (a moot point if you remove this function)
</blockquote></details>

Done.  Good catch.  The extra const makes sense.  Note: I did remove the function.
___
*[`examples/manipulation_station/manipulation_station.h` line 489 at r1](https://reviewable.io/reviews/robotlocomotion/drake/17847#-NBOe2gIB9dOsDMxZgsU:-NBUexc95Y80kEXnQ2U8:b-896fix) ([raw file](https://github.com/robotlocomotion/drake/blob/931fa293488a34e3306d20b3f1493d491e12176b/examples/manipulation_station/manipulation_station.h#L489)):*
<details><summary><i>Previously, jwnimmer-tri (Jeremy Nimmer) wrote…</i></summary><blockquote>

The string value to hard-code in the test would be `"drake/manipulation/models/wsg_50_description/sdf/schunk_wsg_50_no_tip.sdf"`.  To turn that into absolute path, call `FindResourceOrThrow`.

In other words, replicating the first switch-case item within `AddDefaultWsg` from within the implementation.
</blockquote></details>

Done.

**Review by mitiguy — COMMENTED** [review]
+(status: squashing now) 

> *[Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/17847#-:-NBV8Nwf0a_9itflHVAi:bm9o57a)* status: :shipit: complete! all discussions resolved, LGTM from assignees sherm1(platform),xuchenhan-tri(platform) (waiting on @mitiguy)
